dashboard/utils_test.py

```diff
@@ -478,11 +479,31 @@ def test_has_paid_fa_with_final_grade(self):
         key = self.crun_fa.edx_course_key
         assert mmtrack.has_paid(key) is False
         final_grade = FinalGradeFactory.create(user=self.user, course_run=self.crun_fa, course_run_paid_on_edx=True)
-        assert mmtrack.has_paid(key) is True
+        assert mmtrack.has_paid(key) is False
         final_grade.course_run_paid_on_edx = False
         final_grade.save()
         assert mmtrack.has_paid(key) is False
 
+    def test_has_paid_fa_with_final_grade_course_run_paid_on_mm(self):
```

Comment:
these test case names are a little long. can you change these to `test_not_paid_fa_with_course_run_paid_on_edx` and `test_has_paid_fa_with_course_run_paid_on_mm`